templates/thank-you.html

```diff
@@ -46,24 +47,31 @@ <h1 data-qa="title">{{ _("Thank you for completing the {survey_title}").format(
     <a href="https://www.ons.gov.uk/surveys">https://www.ons.gov.uk/surveys</a></p>
   {% endif %}
 
-  {% if content.view_submitted_response.enabled %}
+  {% if content.view_submitted_response.enabled and not content.view_submitted_response.expired %}
     <h1 data-qa="view-submitted-response-title">{{ _("Get a copy of your answers") }}</h1>
+    <p
+      class="ons-u-mb-s">{{ _("You can <a href='{url}' id='view-submitted-response-link'>save or print your answers</a> for your records.").format(url = content.view_submitted_response.url) }}</p>
     {% if not content.view_submitted_response.expired %}
-      <p class="ons-u-mb-s">{{ _("You can <a href='{url}' id='view-submitted-response-link'>save or print your answers</a> for your records.").format(url = content.view_submitted_response.url) }}</p>
-      {% call onsPanel({
-        "id": "view-submitted-response-warning",
-        "type": "warn"
-      }) %}
-        <p>{{ _("For security your answers will only be available to view for 45 minutes") }}</p>
-      {% endcall %}
-    {% else %}
-      {% call onsPanel({
+      {{ onsTimeoutPanel ({
+          "id": "view-submitted-response-countdown",
+          "minutesTextSingular": _("minute"),
+          "minutesTextPlural": _("minutes"),
+          "secondsTextSingular": _("second"),
+          "secondsTextPlural": _("seconds"),
```

Comment:
I find this odd, is it done purely for translation? As I can't think when minutesTextSingular would ever be anything other than minute

Reply:
Ya, it's for translations.

Reply:
As per contract with DS panel/modal, this needs to be there. we don't control this. probably best to raise with DS.